README.md

```diff
@@ -175,3 +175,50 @@ String yburl = "jdbc:yugabytedb://127.0.0.1:5433/yugabyte?user=yugabyte&password
     Otherwise, when the driver needs to connect to any of these additional endpoints (when the primary endpoint
     specified via `serverName` is unavailable), it will use the default port number (`5433`) and not
     `dataSource.portNumber` to connect to them.
+
+## Logging using java.util.logging
+
+The YugabyteDB JDBC Driver supports the use of logging (or tracing) to help resolve issues with the pgJDBC Driver when is used in your application.
+
+The smart Driver uses the logging APIs of java.util.logging that is part of Java since JDK 1.4, which makes it a good choice for the driver since it doesn’t add any external dependency for a logging framework. java.util.logging is a very rich and powerful tool.
```

Comment:
```suggestion
The smart driver uses the logging APIs of `java.util.logging` that is part of Java since JDK 1.4, Also, it's a good choice for the driver since it doesn’t add any external dependency for a logging framework. `java.util.logging` is a very rich and powerful tool.
```